docs/en/cloud/manage/settings.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_label: Configuring Settings
+slug: /en/manage/settings
+---
+
+# Configuring Settings
+
+To specify settings for your ClickHouse Cloud service for a specific [user](/docs/en/operations/access-rights#user-account-management) or [role](/docs/en/operations/access-rights#role-management), you must use [SQL-driven Settings Profiles](/docs/en/operations/access-rights#settings-profiles-management). Applying Settings Profiles ensures that the settings you configure persist, even when your services stop, idle, and upgrade. To learn more about Settings Profiles, please see [this page](/docs/en/operations/settings/settings-profiles.md).
+ 
+Please note that XML-based Settings Profiles and [configuration files](/docs/en/operations/configuration-files.md) are currently not supported for ClickHouse Cloud.
```

Comment:
Here it's mentioned `currently not supported`. I don't know if we intend to support it in the future. Maybe we should say `are not supported`

Reply:
I am ok with "currently" - our plans can always change even if something is not on the immediate roadmap, so it is not inaccurate.